recipes/treesapp/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
 
 build:
   number: 0
-  skip: True  # [not linux64]
+  skip: True  # [osx or py2k]
```

Comment:
Does  this `skip: True  # [osx or py2k]` indicate only the conda package will only be built for linux and python 3? I'd like to support Mac as well if possible.

Reply:
Yes, I had assumed since you had `not linux64` that you only wanted linux support (that's what that means). I'll reenable OSX builds real quick.

Reply:
Perfect, thanks and sorry for yet _another_ silly mistake!. In retrospect, and after reading about preprocessing selectors, I really have no idea why I didn't write `py2k`. I really appreciate your help!